src/views/ManageList.jsx

```diff
@@ -18,6 +21,12 @@ export function ManageList({ listPath }) {
 		setDaysUntilNextPurchase(7);
 	};
 
+	const handleShare = (event) => {
+		event.preventDefault();
+		shareList(listPath, currentUserId, recipientEmail);
```

Comment:
shareList is an async function, so I would consider updating the handler function to reflect this, incase you want to add more logic to the function later on that needs to wait on shareList to finish

Reply:
@jtkabenni Thank you so much for the suggestions I have refactored the implementation by removing the async and await keywords and instead using the `Resolve` and `Reject` state of Promise when calling the shareList function. This approach allows us to handle the response from the Promise more cleanly and move the alert logic out of the firebase.js file. I believe this makes the code more streamlined and easier to read and maintain. Could you please review these changes when you have a moment? 
Also, informing the rest of the team about this update to ensure everyone is aligned. @stacy-tech @dterceroparker @NickRoccodev11